packages/fether-react/src/Tokens/Tokens.js

```diff
@@ -39,12 +69,19 @@ class Tokens extends PureComponent {
       account: { address },
       history
     } = this.props;
+    const { showWarning } = this.state;
 
     const backupAccountItem = {
       name: 'Backup Account',
       onClick: () => history.push(`/backup/${address}`)
     };
 
+    const backupRecoveryItem = {
+      name: 'Backup Recovery',
```

Comment:
Although this feels long, we need to talk about "recovery phrase" here, as "recovery" alone doesn't tell enough. if "Backup" is too long we can consider "Save Recovery phrase"?

Reply:
Agreed, "Save Recovery Phrase" sounds better